front/lib/swr.ts

```diff
@@ -120,6 +120,17 @@ export const postFetcher = async ([url, body]: [string, object]) => {
   return resHandler(res);
 };
 
+type UrlsAndOptions = { url: string; options: object };
+
+const fetcherMultiple = <T>(urlsAndOptions: UrlsAndOptions[]) => {
+  const f = (url: string, options: object) =>
+    fetcher(url, options).then((r) => r);
+
+  return Promise.all<T>(
+    urlsAndOptions.map(({ url, options }) => f(url, options))
```

Comment:
OOC what logic is then being used by the caching system of SWR?

Reply:
I don't know but it seems to work fine as the request is never retriggered.